server/routes/activities/daily-attendance-summary/handlers/applyFilters.ts

```diff
@@ -9,14 +9,20 @@ export class Filters {
   @Expose()
   @Transform(({ value }) => (value ? [value].flat() : undefined)) // Transform to an array if only one value is provided
   activityFilters?: string[]
-
-  @Expose()
-  searchTerm?: string
 }
 
 export default class ApplyFiltersRoutes {
   APPLY = async (req: Request, res: Response): Promise<void> => {
-    const { categoryFilters, activityFilters, searchTerm } = req.body
+    const { categoryFilters, activityFilters } = req.body
+    const searchTermArray = req.body.searchTerm
+    let nonEmptySearchTerm = ``
+    if (searchTermArray) {
+      searchTermArray.forEach((search: string) => {
+        if (search.trim() !== '') {
+          nonEmptySearchTerm = search
+        }
+      })
+    }
```

Comment:
Is searchTerm always a string[] or could it be a string where only one is supplied?  There is a utils method `convertToArray` which can take either a string or an array of strings and always return a string[] if useful here?

Reply:
The search term is included in the tabs so there will always be 4, one per each tab.